doc/api/fs.markdown

```diff
@@ -340,6 +340,12 @@ The file is created if it does not exist.
 
 * `'ax+'` - Like `'a+'` but fails if `path` exists.
 
+On POSIX platforms `flags` can also be a number as documented by open(2);
+constants are available from `require('constants')`.  Opening named pipes (aka
+FIFOs) in `O_NONBLOCK` mode can avoid some waiting, but such fds cannot be used
+with `fs.ReadStream` or `fs.WriteStream` and should be passed to `net.Socket`
+instead.
```

Comment:
This is arguably too arcane / too special interest for this section.  Remember the 'no UNIX trivia' rule?